__tests__/nc-news.test.js

```diff
@@ -0,0 +1,56 @@
+const app = require("../app");
+const request = require("supertest");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+const db = require("../db/connection");
+
+beforeEach(() => {
+    return seed(testData);
+});
+
+afterAll(() => {
+    db.end();
+});
+
+describe("/api/topics", () => {
+    test("GET200: sends an array of topics to the client", () => {
+        return request(app)
+            .get("/api/topics")
+            .expect(200)
+            .then(({ body }) => {
+                expect(body.topics).toEqual([
+                    {
+                        description: "The man, the Mitch, the legend",
+                        slug: "mitch",
+                    },
+                    {
+                        description: "Not dogs",
+                        slug: "cats",
+                    },
+                    {
+                        description: "what books are made of",
+                        slug: "paper",
+                    },
+                ]);
+            });
+    });
+    test("GET200: each topic from response has a slug and a description", () => {
+        return request(app)
+            .get("/api/topics")
+            .expect(200)
+            .then(({ body }) => {
+                body.topics.forEach((topic) => {
+                    expect(topic).toMatchObject({
+                        description: expect.any(String),
+                        slug: expect.any(String),
+                    });
+                });
+            });
```

Comment:
These two tests are both covering the same things, if I had to pick one I'd say the second is only one you really need as it's using fewer lines and is more expansive ( not that we should expect test data to change )